invenio_app_ils/mail/views.py

```diff
@@ -0,0 +1,69 @@
+# Copyright (C) 2021 CERN.
+#
+# invenio-app-ils is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Invenio App ILS email views."""
+
+from flask import Blueprint
+from webargs import fields
+from webargs.flaskparser import use_kwargs
+
+from invenio_app_ils.mail.model import EmailLog
+from invenio_app_ils.permissions import need_permissions
+
+
+def serialize_email(email):
+    """Serialize email from db."""
+    if email is None:
+        return {}
+    email = email.__dict__
+    del email["_sa_instance_state"]
```

Comment:
question: is the date of the triggered_on serialized back correctly is ISO format?

Reply:
The date has the format `"Mon, 22 Feb 2021 16:15:17 GMT"`

Reply:
we should always store dates in the ISO format to avoid mistakes

Reply:
I added a line above to transform it to ISO format.